### PR TITLE
chore: support minitest 5.23.0

### PR DESCRIPTION
**Pull request by seuros**
Added gemfile for 5.23.0

### CODE REVIEW

**Comment by grosser** [comment]
thx looks great!

File: gemfiles/523.gemfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+gemspec :path=>"../"
+gem "minitest", "~> 5.19.0"
```
Comment (grosser) [inline gemfiles/523.gemfile:3]: ah this was wrong, making a fixup

File: maxitest.gemspec [hunk 2]
```diff
@@ -9,7 +9,7 @@ Gem::Specification.new "maxitest", Maxitest::VERSION do |s|
   s.license = "MIT"
   s.executables = ["mtest"]
 
-  s.add_runtime_dependency "minitest", [">= 5.14.0", "< 5.21.0"]
```
Comment (grosser) [inline maxitest.gemspec:12]: this was wrong too, needs to be <5.24

File: gemfiles/523.gemfile (hunk 1 above)
Comment (seuros) [reply gemfiles/523.gemfile:3]: Will fix in 2 minutes

**Comment by grosser** [comment]
https://github.com/grosser/maxitest/pull/62

File: gemfiles/523.gemfile (hunk 1 above)
Comment (grosser) [reply gemfiles/523.gemfile:3]: already got a PR ... trying to look into the CI failure next

**Pull request merged**